packages/node/src/ethereum/uninstall-commitment.ts

```diff
@@ -44,11 +41,11 @@ export class UninstallCommitment extends MultiSendCommitment {
       to: this.networkContext.NonceRegistry,
       value: 0,
       data: nonceRegistryIface.functions.setNonce.encode([
-        0, // Timeout is 0 for dependencyNonce!
-        keccak256(defaultAbiCoder.encode(["uint256"], [this.dependencyNonce])),
-        // Hard coded the update to 1 because that is the value
-        // that represents an app as being "uninstalled"
-        DependencyValue.UNINSTALLED
+        /* timeout */ 0,
+        /* salt */ keccak256(
+          defaultAbiCoder.encode(["uint256"], [this.dependencyNonce])
+        ),
+        /* nonceValue */ DependencyValue.CANCELLED
```

Comment:
Hey @ricmoo is there any way `encode` could take an object with named arguments so we didn't have to do this weird commenting pattern as you see above?